release-check.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
maybe use #!/usr/bin/env bash (http://stackoverflow.com/questions/16365130/the-difference-between-usr-bin-env-bash-and-usr-bin-bash)